************* Module gevent._sslgte279
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:632:0: E0012: Bad option value 'arguments-renamed' (bad-option-value)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:637:0: E0012: Bad option value 'arguments-renamed' (bad-option-value)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:18:7: W0212: Access to a protected member _ssl of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:20:0: C0413: Import "import errno" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:21:0: C0413: Import "from gevent._socket2 import socket" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:22:0: C0413: Import "from gevent._socket2 import AF_INET" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:23:0: C0413: Import "from gevent.socket import timeout_default" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:24:0: C0413: Import "from gevent.socket import create_connection" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:25:0: C0413: Import "from gevent.socket import error as socket_error" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:26:0: C0413: Import "from gevent.socket import timeout as _socket_timeout" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:27:0: C0413: Import "from gevent._compat import PYPY" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:28:0: C0413: Import "from gevent._util import copy_globals" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:61:23: W0212: Access to a protected member _fileobject of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:63:0: R0205: Class '_fileobject' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:74:0: C0103: Class name "orig_SSLContext" doesn't conform to PascalCase naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:80:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:139:0: R0913: Too many arguments (9/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:197:33: W0622: Redefining built-in 'type' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:193:4: R0913: Too many arguments (18/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:265:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:288:12: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:306:4: C0103: Method name "_checkClosed" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:318:19: W0622: Redefining built-in 'len' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:463:16: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:465:12: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:570:8: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:572:12: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:616:23: W0212: Access to a protected member _wrap_socket of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:619:16: C0103: Variable name "rc" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:621:16: C0103: Variable name "rc" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:648:8: W0212: Access to a protected member _drop_events_and_close of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/site-packages/gevent/_sslgte279.py:707:0: R0913: Too many arguments (10/5) (too-many-arguments)

-----------------------------------
Your code has been rated at 9.07/10

